docs/changelog.rst

```diff
@@ -2,8 +2,15 @@
 Changelog
 =========
 
+Unreleased
+----------
+
 * Update the :ref:`migration CLI <migration-cli>` to detect unittest classes based on whether they use ``self.assert*`` methods like ``self.assertEqual()``.
 
+* Fix free-threaded Python warning: ``RuntimeWarning: The global interpreter lock (GIL) has been enabled...`` as seen on Python 3.13+.
```

Comment:
I improved this to include part of the warning message, to help readers see if it's the a warning they encountered.